### ISSUE TITLE
How to use [imageSize]

### ISSUE THREAD

**Issue by azisn11s** [sanjayV/ng-image-slider#1]
Hi, I was wondering how to override each image width.

```
<ng-image-slider 
      [images]="imageObject" 
      [infinite]="true" 
      [slideImage]="2" 
      [imageSize]="400">
    </ng-image-slider>
```

But, the images still rectangle. If there is another way to override via CSS, please reply with example.

**Comment by sanjayV** [comment]
@azisn11s I have fixed image size issue in new version (**1.0.4**)
Use following structure for set slider image width or height.

`[imageSize]="{width:` 400, height: 300}"
                          OR
if you just want to set width then you can use
`[imageSize]="{width: 400}"`

**Comment by Naman9871** [comment]
how can we add different href at every different pic of ng-image-slider   ?
